e2e/nested-router-tab-view/app/app.component.html

```diff
@@ -1 +1 @@
-<page-router-outlet></page-router-outlet>
\ No newline at end of file
+<page-router-outlet tag="rootPRO"></page-router-outlet>
```

Comment:
Is the `tag="rootPRO"` used somewhere or just for debugging?

Reply:
it is just for debugging, since it is very easy to distinguish which p-r-o is activated, states and so on..